monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/home/NavMenuItems.js

```diff
@@ -42,96 +43,126 @@ import LayersIcon from '@material-ui/icons/Layers';
-            <ListItemText primary="Endpoints" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <InbountEpIcon />
-            </ListItemIcon>
-            <ListItemText primary="Inbound Endpoints" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <MessageProcessorIcon />
-            </ListItemIcon>
-            <ListItemText primary="Message Processors" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <MessageStoreIcon />
-            </ListItemIcon>
-            <ListItemText primary="Message Stores" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <ApiIcon />
-            </ListItemIcon>
-            <ListItemText primary="API" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <TemplateIcon />
-            </ListItemIcon>
-            <ListItemText primary="Templates" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <SequenceIcon />
-            </ListItemIcon>
-            <ListItemText primary="Sequences" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <Event />
-            </ListItemIcon>
-            <ListItemText primary="Tasks" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <LocalEntriesIcon />
-            </ListItemIcon>
-            <ListItemText primary="Local Entries" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <DssIcon />
-            </ListItemIcon>
-            <ListItemText primary="Data Services" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <LayersIcon />
-            </ListItemIcon>
-            <ListItemText primary="Datasources" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <ConnectorIcon />
-            </ListItemIcon>
-            <ListItemText primary="Connectors" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <CappIcon />
-            </ListItemIcon>
-            <ListItemText primary="Carbon Applications" />
-        </ListItem>
-        <ListItem button>
-            <ListItemIcon>
-                <LogFileIcon />
-            </ListItemIcon>
-            <ListItemText primary="Log Files" />
-        </ListItem>
+        <Link to={'/proxy_services'} style={{textDecoration: 'none', color: 'rgba(0, 0, 0, 0.87)'}}>
+            <ListItem button>
+                <ListItemIcon>
+                    <ProxyIcon/>
+                </ListItemIcon>
+                <ListItemText primary="Proxy Services"/>
+            </ListItem>
+        </Link>
+        <Link to={'/endpoints'}  style={{ textDecoration: 'none' , color: 'rgba(0, 0, 0, 0.87)'}}>
```

Comment:
Can't we use a css class instead of inline styles in these places `<Link to={'/proxy_services'} style={{textDecoration: 'none', color: 'rgba(0, 0, 0, 0.87)'}}>` 
With that, we can avoid duplicating CSS attributes and values.

Reply:
The same can be done in several other places